internal/models/todo.go

```diff
@@ -0,0 +1,27 @@
+package models
+
+import (
+	"time"
+)
+
+type Todo struct {
+	ID            int       `json:"id"`
+	Description   string    `json:"description"`
+	DateCreated   time.Time `json:"date_created"`
+	DateUpdated   time.Time `json:"date_updated"`
+	DateCompleted time.Time `json:"date_completed"`
+	Completed     bool      `json:"completed"`
+	UserID        int       `json:"user_id"`
+}
+
+var TodoList []Todo
+var TodoID = 1
+
+type TodoCreateRequest struct {
```

Comment:
make a directory named `http`. Under `http` make two directories named `rq` and `rs` for API request and response models and move all request and response models there